app/models/manageiq/providers/cisco_intersight/inventory/collector/physical_infra_manager.rb

```diff
@@ -3,133 +3,147 @@ class Inventory::Collector::PhysicalInfraManager < ManageIQ::Providers::Inventor
 
     def collect
 
-      set_configuration
-
+      # Establish conneciton. Connection in inside ManagerMixin sets API key and keyid-d
```

Comment:
```suggestion
      # Establish connection. Connection in inside ManagerMixin sets API key and keyid-d
```

Reply:
Thanks for the suggestion, did not notice this typo :)